source/fundamentals/authentication/mechanisms.txt

```diff
@@ -235,6 +235,22 @@ The driver checks for your credentials in the following sources in order:
       .. literalinclude:: /code-snippets/authentication/aws-env-variable.js
          :language: javascript
 
+.. important:: Retrieval of AWS Credentials
+   
+   Starting in version 4.11, when you install the optional
+   ``aws-sdk/credential-providers`` dependency, the driver uses the AWS SDK
+   to retrieve credentials from the environment. As a result, if you
+   have a shared AWS credentials file or config file, the driver will
+   use those credentials by default if there are no AWS authentication
+   environment variables set.
+   
+   To override this behavior, you can perform one of the following actions:
```

Comment:
Suggestion

I think this sentence is a little long and covers a couple of ideas, so I would recommend breaking it into two different sentences and combining it with the next line. I also reworded the second sentence a bit to put the actionable part first.

```suggestion
   to retrieve credentials from the environment. As a result, if you
   have a shared AWS credentials file or config file, the driver will
   use those credentials by default. You can override this behavior
   by performing one of the following actions:
```